### DISCUSSION TITLE
Transfer cosmic-ctl to cosmic-utils

### DISCUSSION THREAD

**Discussion by HeitorAugustoLN** [cosmic-utils/discussions#4] [General]
Hi everyone,  

I’d like to propose moving my project [cosmic-ctl](https://github.com/HeitorAugustoLN/cosmic-ctl) to the `cosmic-utils` organization.  

`cosmic-ctl` is a command-line tool for managing COSMIC Desktop settings. It lets users:  
- Read, write, and delete settings  
- Backup configurations
- Apply configurations

By transferring it, the project could reach more users and get more community support. It would also fit well with the goal of `cosmic-utils` to host useful tools for COSMIC.  

Let me know what you think!  

Thanks,  
Heitor

**Comment by edfloreshz** [comment]
I'm interested in your project and would like to ask if you'd be willing to integrate it into Tweaks?

I feel that it would be a good fit for the snapshots page.

**Reply by HeitorAugustoLN** [reply]
Hi @edfloreshz,

Thanks for reaching out and for your interest in cosmic-ctl! Integrating it into Tweaks, especially for the snapshots page, sounds like a good idea.

Right now, I’ve been focused on keeping it as a standalone CLI tool, but I’m definitely open to discussing how it could work with Tweaks. Do you have any specific thoughts on how the integration might look?

**Reply by edfloreshz** [reply]
I was thinking of providing the same functionality the CLI provides in a GUI. 

Tweaks is able to make snapshots of the current desktop layout and restore them at any point, but being able to make snapshots of everything else would be awesome.

**Reply by HeitorAugustoLN** [reply]
That sounds awesome! Adding full config snapshots to Tweaks would be a great feature. `cosmic-ctl` could definitely handle the backend for it while Tweaks provides the GUI.

I am currently doing a big refactor in `cosmic-ctl` to provide an interactive terminal user interface too, so I might take some time to integrate it there. I also just submitted a PR in Tweaks, that adds a nix development environment, so I can easily develop the program in there

**Reply by edfloreshz** [reply]
Merged, I'll invite you into the org so you can move the project there.

**Reply by HeitorAugustoLN** [reply]
I transferred the repo, but I think somehow transferred it without commit acess for me.

**Reply by edfloreshz** [reply]
Please check again, should be fixed

**Reply by HeitorAugustoLN** [reply]
I got the access again, thanks!

**Reply by HeitorAugustoLN** [reply]
@edfloreshz sorry for the ping, but could you enable "Allow GitHub Actions to create and approve pull requests" in the organization, when possible?
Because I am trying to enable it for my repository, but it looks like it needs to be enabled at the organization level first.

**Reply by edfloreshz** [reply]
Should be done

**Reply by HeitorAugustoLN** [reply]
Thanks :)
